src/main/scala/com/socrata/tickertape/TickerTapeCLI.scala

```diff
@@ -21,12 +21,13 @@ object TickerTapeCLI extends StrictLogging {
     })
 
     while (true) {
-      writeMetricsBatch(queue)
 
-      // Closing the MetricQueue object causes a metric file to be completed,
-      // which is necessary before balboa-agent will process it.
-      queue.close()
-      queue = BalboaConfig(config.balboa).queue
+      try {
+        writeMetricsBatch(queue)
+      } finally {
+        queue.close
```

Comment:
Personally, I think leaving parens off of 0-ary function/method calls in Scala is terribly inconsistent with other calls, but that's just an opinion and we do this all over the place.